rest/api_test.go

```diff
@@ -2601,6 +2601,115 @@ func TestBulkGetRevPruning(t *testing.T) {
 
 }
 
+
+
+// Reproduces panic seen in https://github.com/couchbase/sync_gateway/issues/2528
+func TestBulkGetBadAttachmentReproIssue2528(t *testing.T) {
+
+	var rt RestTester
+	var body db.Body
+
+	// Disable rev cache so that the _bulk_get request is forced to go back to the bucket to load the doc
+	// rather than loading it from the (stale) rev cache.  Required when testing against a walrus bucket.
```

Comment:
Updated.  When running against a Couchbase bucket it wasn't necessary to touch `KDefaultRevisionCacheCapacity`.  I assumed it was because something on the DCP feed was updating the revision cache, but I didn't dig into it further.